cmd/protoc-gen-openapi/plugin_test.go

```diff
@@ -258,3 +259,40 @@ func TestOpenAPIDefaultResponse(t *testing.T) {
 		})
 	}
 }
+
+func TestOpenAPIStringInteger64(t *testing.T) {
+	for _, tt := range openapiTests {
+		fixture := path.Join(tt.path, "openapi_string_integer64.yaml")
+		if _, err := os.Stat(fixture); errors.Is(err, os.ErrNotExist) {
+			if !GENERATE_FIXTURES {
```

Comment:
Doe this have any relevance to `gnostic`? Or is it something used locally?

Reply:
sorry, I don't understand, I just copy from other unit tests

Reply:
Sorry. I thought `GENERATE_FIXTURES` was new.